Configuration/StandardSequences/python/Services_cff.py

```diff
@@ -10,7 +10,9 @@
 
 # load CUDA services when the "gpu" or "pixelNtupletFit" modifiers are enabled
 def _addCUDAServices(process):
-     process.load("HeterogeneousCore.CUDAServices.CUDAService_cfi")
+    process.load("HeterogeneousCore.CUDAServices.CUDAService_cfi")
+    if 'MessageLogger' in process.__dict__:
+        process.MessageLogger.CUDAService = cms.untracked.PSet()
```

Comment:
@makortel, in the meantime I think this could be a reasonable workaround ?
```suggestion
    if not 'MessageLogger' in process.__dict__:
        process.load("FWCore.MessageService.MessageLogger_cfi")
    process.MessageLogger.CUDAService = cms.untracked.PSet()
```

Reply:
It should work (actually unconditional `process.load()` should work as well because the cfi and the `process` have the same `MessageLogger` object). I have also a PR for the change in framework in https://github.com/cms-sw/cmssw/pull/35298.